app/code/Magento/InventoryExportStockApi/Test/Api/ExportStockIndexDataTestStockTestWebsiteReservationsTest.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryExportStockApi\Test\Api;
+
+use Magento\Framework\ObjectManager\ObjectManager;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+use Magento\Framework\Webapi\Rest\Request;
+use Magento\Catalog\Api\Data\ProductInterface;
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Quote\Api\CartManagementInterface;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Magento\Quote\Api\Data\CartInterface;
+use Magento\Quote\Api\Data\CartItemInterface;
+use Magento\Quote\Api\Data\CartItemInterfaceFactory;
+use Magento\Sales\Api\OrderManagementInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\Quote\Api\Data\AddressInterface;
+use Magento\Quote\Api\Data\AddressInterfaceFactory;
+use Magento\Store\Model\StoreRepository;
+use Magento\Store\Model\StoreManagerInterface;
+
+/**
+ * {@inheritDoc}
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ * @see https://app.hiptest.com/projects/69435/test-plan/folders/908874/scenarios/3042336
+ */
+class ExportStockIndexDataTestStockTestWebsiteReservationsTest extends WebapiAbstract
+{
+    const API_PATH = '/V1/inventory/dump-stock-index-data';
+    const SERVICE_NAME = 'inventoryExportStockApiExportStockIndexDataV1';
+    const EXPORT_PRODUCT_COUNT = 7;
+
+    /**
+     * @var ObjectManager
+     */
+    private $objectManager;
+
+    /**
+     * @var ExportStockIndexDataOnCustomStockTest
+     */
+    private $parent;
+
+    /**
+     * @inheritDoc
+     */
+    public function setUp()
+    {
+        $this->objectManager = Bootstrap::getObjectManager();
+        $this->parent = $this->objectManager->create(ExportStockIndexDataOnCustomStockTest::class);
```

Comment:
This has to be refactored. It's not a common practice to inject one test into another